Core/src/org/sleuthkit/autopsy/report/ReportGenerator.java

```diff
@@ -145,9 +138,17 @@ public void windowClosing(WindowEvent e) {
      */
     void generateGeneralReport(GeneralReportModule generalReportModule) {
         if (generalReportModule != null) {
+            reportPathFormatString = String.format(reportPathFormatString, generalReportModule.getName());
+            // Create the root reports directory.
+            try {
+                FileUtil.createFolder(new File(reportPathFormatString));
+            } catch (IOException ex) {
+                errorList.add(NbBundle.getMessage(this.getClass(), "ReportGenerator.errList.failedMakeRptFolder"));
+                logger.log(Level.SEVERE, "Failed to make report folder, may be unable to generate reports.", ex); //NON-NLS
+            }
```

Comment:
Moving the folder creation from the constructor to the various methods highlights a problem with this approach.  If there was a problem creating the directory, there is no point in calling into the ReportModule(s). I want to move this code change into the code base, but I will create a follow up story in JIRA for addressing this.